### PR TITLE
Release v7.0.4

### PR DESCRIPTION
**Pull request by priesgo**
#### Major changes

* CVA data intake models for pedigrees and cancer participant

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -11,7 +11,7 @@
     <packaging>${p.type}</packaging>
 
     <properties>
-        <models.version>7.0.3</models.version>
+        <models.version>6.1.5</models.version>
```
Comment (kevinpetersavage) [inline pom.xml]: Is this right?

Comment (priesgo) [reply pom.xml]: it is right, in Java we are still stuck with build `6.1` and remember that Java packaging does not support the different versions

Comment (priesgo) [reply pom.xml]: (I know, it is horrible)

**Comment by greglever** [comment]
@priesgo I'd recommend squashing the commits to a signed commit because:
```
Merging is blocked
The target branch requires all commits to be signed. 
```

**Pull request merged**